src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -147,4 +173,27 @@ class KeyPhraseExtractionSuiteV4 extends TestBase with DataFrameEquality with Te
         row.toSeq.foreach { col => println(col) }
       }
     }
+      test("KPE - Assert Model Version") {
+        val outputCol = extractor.transform(df2)
+          .select("output")
+          .collect()
+        assert(outputCol(0).schema(0).name == "output")
+        val fromRow = KeyPhraseResponseV4.makeFromRowConverter
+        outputCol.foreach(row => {
+          val outResponse = fromRow(row.getAs[GenericRowWithSchema]("output"))
+          assert(outResponse.modelVersion.get == "2021-06-01")
+        })
+      }
+  test("KPE - Statistics Assertion") {
+    val outputCol = extractor.transform(df2)
+      .select("output")
+      .collect()
+    assert(outputCol(0).schema(0).name == "output")
+    val fromRow = KeyPhraseResponseV4.makeFromRowConverter
+    outputCol.foreach(row => {
+      val outResponse = fromRow(row.getAs[GenericRowWithSchema]("output"))
+      assert(outResponse.modelVersion.get == "2021-06-01")
+      assert(outResponse.statistics.indexOf(0) == 11)

```

Comment:
What are we asserting here? Can you assert the values of valid docs, erroneous docs, etc?